_haikus/natural_cloud

```diff
@@ -0,0 +1,11 @@
+---
+layout: haiku
+title: Natural Cloud
+author: SNM
+---
+
+I wandered lonely as beginner<br/>
+When all at once I saw a Cloud,<br/>
+A host, of golden code; <br/>
+Beside the work, beneath the knowledge,<br/>
+Joyfully Learning<br/>
```

Comment:
Please read the section about haikus in the `README.md` file of this repository. Haikus are poems that follow a specific format of three lines. They also have a certain number of syllables per line: 5 syllables in the first line, 7 syllables in the second line, 5 syllables in the last line. You can check your poem against haiku standards with [http://www.haikusyllablecounter.com/](http://www.haikusyllablecounter.com/).